terraform/aks/config/production_Terrafile

```diff
@@ -1,3 +1,3 @@
 aks:
     source:  "https://github.com/DFE-Digital/terraform-modules"
-    version: "stable"
+    version: "v0.21.0-alpha"
```

Comment:
change back to stable?

Reply:
this was done in another PR

Reply:
well won't this PR change it back to v0.21.0-alpha unless you remove it